include/openthread/thread_ftd.h

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2016, The OpenThread Authors.
+ *  Copyright (c) 2017, The OpenThread Authors.
```

Comment:
nit: Should this be: 2016 - 2017 reflecting that there is/was a copyright in 2016?